src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchProfileExtensions.cs

```diff
@@ -8,6 +8,7 @@ internal static class LaunchProfileExtensions
         public const string SqlDebuggingProperty = "sqlDebugging";
         public const string RemoteDebugEnabledProperty = "remoteDebugEnabled";
         public const string RemoteDebugMachineProperty = "remoteDebugMachine";
+        public const string RemoteAuthenticationModeProperty = "remoteAuthenticationMode";
```

Comment:
UWP and Legacy calls this "AuthenticationMode" can we pick the same name "authenticationMode" so that it makes easier to rationalize when we have to rationalize their settings with ours when we replace legacy?